### PR TITLE
Fix issue that deepTools give error when handling single-file input

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Comment by chuan-wang** [comment]
Tested and worked as expected.

File: main.nf [hunk 1]
```diff
@@ -440,33 +440,36 @@ process deepTools {
...
+        plotCorrelation \\
+            -in multiBamSummary.npz \\
+            -o heatmap_SpearmanCorr_multiBamSummary.png \\
+            --corMethod spearman \\
+            --skipZeros \\
+            --plotTitle "Spearman Correlation of Read Counts" \\
+            --whatToPlot heatmap \\
+            --colorMap RdYlBu \\
+            --plotNumbers
+    fi
```
Comment (ewels) [inline main.nf]: One small thing, could we get this logic printed to the log? Will make debugging easier. For example:
```bash
else
    echo "Only one BAM input file found. Skipping multiBam commands."
fi
```

**Comment by ewels** [comment]
I prefer having this logic within the Nextflow script if I'm honest, as we can have better logging (direct to the nextflow log, rather than having to look inside work directories), plus the bash scripts generated don't contain any runtime logic so are easier to debug. But if this works, then I'm happy for now (though would be good to add extra log line as suggested).

**Comment by chuan-wang** [comment]
Good idea! I will test it

**Comment by ewels** [comment]
Great stuff, thanks!

**Pull request merged**